File: trunk/src/main/java/net/esper/eql/expression/ExprValidationException.java
package net.esper.eql.expression;

/**
 * Thrown to indicate a validation error in a filter expression.
 */
public class ExprValidationException extends Exception
{
    /**
     * Ctor.
     * @param message - validation error message
     */
    public ExprValidationException(String message)
    {
        super(message);
    }
}

File: NEsper/eql/expression/ExprValidationException.cs
using System;
namespace net.esper.eql.expression
{
	
	/// <summary> Thrown to indicate a validation error in a filter expression.</summary>
	[Serializable]
	public class ExprValidationException:System.Exception
	{
		/// <summary> Ctor.</summary>
		/// <param name="message">- validation error message
		/// </param>
		public ExprValidationException(String message):base(message)
		{
		}
	}
}